build/BUILD

```diff
@@ -0,0 +1,56 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_kubernetes_build//defs:build.bzl", "release_filegroup")
+load("@io_kubernetes_build//defs:pkg.bzl", "pkg_tar")
+
+# Our build_tar is much faster at gzip compression but doesn't handle duplicate
+# files correctly. We can remove the double hop once that's fixed.
+# https://github.com/kubernetes/repo-infra/issues/98
+load(
+    "@bazel_tools//tools/build_defs/pkg:pkg.bzl",
+    rpkg_tar = "pkg_tar",
```

Comment:
I'm confused. Won't these two rules map to the same thing?

Reply:
The "real" pkg_tar is assigned to the name "rpkg_tar" and the fast pkg_tar is just pkg_tar. I'll just use the slow, correct one until https://github.com/kubernetes/repo-infra/pull/99 merges.